src/linker/configuration.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 from typing import Dict, List, Optional, Union
 
-from linker.utilities.general_utils import load_yaml
+from linker.utilities.data_utils import load_yaml, validate_input
```

Comment:
I think `validate_input` should be a property of the PipelineSchema class, but that doesn't have to happen in this PR.

Reply:
Agreed. I'd say though that it actually should be part of this PR?

Reply:
done